### ISSUE TITLE
Entity - Service

### ISSUE THREAD

**Issue by yzummy** [kevinschoonover/jeeves#19]
- [x] Service ID
- [x] User ID
- [x] Table ID
- [x] Service Type (Add water, etc)

**Comment by christopher-o-toole** [comment]
Currently the supported service types are:
```
export enum serviceType {
  REFILL_DRINK = 'refill_drink',
  SERVE_FOOD = 'serve_food',
  TAKE_ORDER = 'take_order',
  CHECK_ON_TABLE = 'check_on_table',
  ALLERGEN_ASSISTANCE = 'allergen_assistance',
  UNKNOWN = 'unknown',
}

```

We may add or remove some of these types as needed however.

**Comment by kevinschoonover** [comment]
Closed in #35
